App.js

```diff
@@ -9,8 +9,31 @@ import Calendar from './components/calendar/Calendar';
 import Home from './components/home/Home';
 import Profile from './components/profile/Profile';
 import Workouts from './components/workouts/Workouts';
+import Cycles from './components/cycles/Cycles';
 import { configureFonts, DefaultTheme, Provider as PaperProvider } from 'react-native-paper';
-
+import { Entypo } from '@expo/vector-icons';
+import { AppLoading } from 'expo';
+import {
+  useFonts,
```

Comment:
Not a Huge Deal, but we should get rid of fonts we wont use as each font slightly increase the time the app takes to load. (Like we probably arent ever going to use the italic ones or the super thin and thick ones)

Reply:
Yeah good point, should we just leave them all for now and throw a task up for once we have design finalized to remove extras?

Reply:
Yeah agreed, let's make that a new task and edit out the ones we won't use later on, it isn't very urgent.